frontend/packages/client/src/components/ProposalsList/ProposalList.sass

```diff
@@ -1,45 +1,89 @@
+.blockies-wrapper
+  position: relative
+  background-color: white
+  border-radius: 133.333px
+  max-height: 40px
 .blockies
-  border-radius: 50px
+  border-radius: 133.333px
+  border: 4px solid #F2F2F2
 .rounded-top
   border-top-left-radius: 8px
   border-top-right-radius: 8px
 .proposal-title
-  color: #000000 
-  font-weight: 400 
+  color: #000000
+  font-weight: 400
   font-size: 20px
   font-height: 24px
+.community-wrapper
+  padding-top: 0
+.community-header-wrapper
+  background-color: #F2F2F2
+  width: 100%
+  max-height: 248px
+.community-header
+  justify-content: space-between // here because bulma class didn't override
+.community-logo-wrapper
+  margin: 0
+.community-logo-mobile
+  border-radius: 50%
+  max-height: 85px
+  max-width: 85px
+.community-logo
+  border-radius: 50%
+.community-info
+  padding: 19.5px 24px
+  max-width: 225px
+.community-name
+  font-family: Arimo
+  font-size: 20px
+  font-weight: bold
```

Comment:
if you don't define font-family it works since Aritmo is de primary: `$family-primary: "Arimo"`
font-size 20px can be replaced by `is-size-5` from bulma.
font-weight: bold can be replaced by `has-text-weight-bold` from bulma